app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayFragment.kt

```diff
@@ -278,6 +278,10 @@ class TabsTrayFragment : AppCompatDialogFragment() {
             owner = this,
             view = view
         )
+
+        if (requireArguments().getBoolean(EXTRA_MULTI_SELECT)) {
```

Comment:
Does this check need to be done here or can it go into the `initialState` when creating the `TabsTrayStore` on line 96?

Reply:
Hmm... I'll run a few tests and see. I added it last as a precaution.

Reply:
What's nice about using Kotlin flow's and `ifChanged` is that we get the initial state _and_ the changed results from our data source.

This means we don't have to worry about timing issues (assuming our features are not dependant on each other).

Reply:
Sorry for the delay, got caught up yesterday with another PR. 
Yeah, I agree. Will update this asap.